saiku-ui/js/saiku/views/SelectionsModal.js

```diff
@@ -405,7 +405,7 @@ var SelectionsModal = Modal.extend({
 
             // Filter out used members
             this.available_members = _.select(this.available_members, function(o) {
-                return used_members.indexOf(o.obj.caption) === -1;
+                return used_members.indexOf(o.caption) === -1;
```

Comment:
The correct is `return used_members.indexOf(o.obj ? o.obj.caption : o.caption) === -1;`